ext/reflection/php_reflection.c

```diff
@@ -1790,6 +1790,20 @@ ZEND_METHOD(ReflectionFunctionAbstract, isUserDefined)
 }
 /* }}} */
 
+/* {{{ Returns whether this function is an anonymous closure or not */
+ZEND_METHOD(ReflectionFunction, isAnonymous)
+{
+	reflection_object *intern;
+	zend_function *fptr;
+
+	if (zend_parse_parameters_none() == FAILURE) {
+		RETURN_THROWS();
+	}
```

Comment:
```suggestion
	ZEND_PARSE_PARAMETERS_NONE();
```
This is the shorter and more modern way :-)

Reply:
updated thanks :)